src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -930,6 +930,22 @@ def has_bluestore_label(device_path):
 
     return isBluestore
 
+def has_seastore_label(device_path: str) -> bool:
+    is_seastore = False
+    seastore_disk_signature = b'seastore block device\n'  # 23 bytes including newline
```

Comment:
depends on https://github.com/ceph/ceph/pull/64333

Reply:
the `has_seastore_label()` function is not used anywhere in ceph-volume yet, it has been implemented in anticipation of potential future use.

Reply:
Can we add the the usage of seastore in this PR as well? Are there any missing deps?